maps/southern_cross/events/wildlife_encounter.dm

```diff
@@ -0,0 +1,94 @@
+// This event sends random sif mobs at someone in the wilderness, unannounced.
+
+/datum/event2/meta/wildlife_encounter
+	name = "random encounter"
+	departments = list(DEPARTMENT_EVERYONE)
+	chaos = 10
+	event_type = /datum/event2/event/wildlife_encounter
+
+/datum/event2/meta/wildlife_encounter/get_weight()
+	var/explorers = metric.count_people_with_job(/datum/job/explorer) + metric.count_people_with_job(/datum/job/sar)
+
+	if(!explorers)
+		return 0
+	return 20 + explorers * 50
+
+/datum/event2/event/wildlife_encounter
+	var/mob/living/victim = null
+
+/datum/event2/event/wildlife_encounter/set_up()
+	var/list/potential_victims = list()
+	for(var/mob/living/L in player_list)
+		//if(!(L.z in get_location_z_levels()))
+		//	log_debug("Not on the right z-level")
+		//	continue // Not on the right z-level.
+
+		if(L.stat)
+			continue // Don't want dead people.
+		if(istype(get_turf(L), /turf/simulated/floor/outdoors) && istype(get_area(L),/area/surface/outside/wilderness))
+			potential_victims += L
+			log_debug("Found victim")
```

Comment:
Not sure I, as a dev who often has debug logs turned on, need to see 7 lines of this output to chat when this event fires because there happens to be as many people on the planet. Also not sure how useful this piece of information would be in server logs after the fact.
Other debug logs in here are probably fine in that regard, but you should probably prefix them with something to indicate what is causing the log

Reply:
The GM outputs what event just fired in the line above.
Should show this now.

DEBUG: Event /datum/event2/event/wildlife_encounter is now being ran. 
DEBUG: Sending 3 /mob/living/simple_mob/animal/sif/shantaks after Michaela Karkee. 
DEBUG: Event /datum/event2/event/wildlife_encounter has finished.